keps/sig-network/3705-cloud-node-ips/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: Cloud Dual-Stack --node-ip Handling
+kep-number: 3705
+authors:
+  - "@danwinship"
+owning-sig: sig-network
+participating-sigs:
+  - sig-node
+  - sig-cloud-provider
+status: provisional
+creation-date: 2022-11-29
+reviewers:
+  - "@mdbooth"
+  - "@khenidak"
+  - "@thockin"
+  - TBD
```

Comment:
@andrewsykim or @cheftako one of you from cloud-provider?

Reply:
Sorry I'm late on this, but feel free to add me as reviewer/approver when KEP goes implementable